core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -564,7 +563,7 @@ class LogManagerTest {
   @Test
   def testBrokerConfigChangeDeliveredToAllLogs(): Unit = {
     logManager.shutdown()
-    val spyConfigRepository = spy(new CachedConfigRepository)
+    val spyConfigRepository = spy(new MockConfigRepository)
```

Comment:
Do we need to "spy" on these instances? Seems like we could put whatever test-only behavior we want into the MockConfigRepository class itself. Or maybe we need to do this for some Mockito reason like "all arguments must be mock/spy-ed"

Reply:
we want to verify that `topicConfig` was called, and this seemed like the easiest way.